src/db/migrations/20230618104332_feature_flags.js

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ export function up (knex) {
+  return knex.schema
+    .createTable('feature_flags', table => {
+      table.string('name').primary().unique()
+      table.boolean('is_enabled').defaultTo(false)
+      table.text('description').default('')
+      table.specificType('dependencies', 'character varying(255)[]')
```

Comment:
I'm not sure I remember what the values of these were, but should these be [`json`](https://knexjs.org/guide/schema-builder.html#json) or [`jsonb`](https://knexjs.org/guide/schema-builder.html#jsonb) columns? Or is the `[]` a cool Postgres feature I haven't heard of yet that allows you to store arrays in columns?

Reply:
Yea it's a postgres specific type that Knex doesn't support (fully).. Though this is a recommended way of using it with Knex